lib/cache.js

```diff
@@ -23,7 +23,7 @@ class Cache {
   getPage(page) {
     let preferredPlatform = platform.getPreferredPlatformFolder(this.config);
     let preferredLanguage = 'en';
-    if (process.env['LANG'] !== '') {
+    if (process.env['LANG'] !== undefined && process.env['LANG'] !== '') {
```

Comment:
1. `process.env` holds dictionary of strings so it can be either a string, an empty string or `undefined`
2. because both empty string and `undefined` are falsey values you are supposed to check it in usual and most logical way:
```js
if (process.env['LANG']) {
  preferredLanguage = process.env['LANG'];
}
```
3. There isn't something magical about uppercase variable names in javascript so usual way of _dot access_ should be used: `process.env.LANG` 
4. This boils down to using `LANG` or predefined fallback which is basically very definition of OR assignment:
```js
let preferredLanguage = process.env.LANG || 'en';
```
5. And because it is never mutated it should be also marked as `const` which brings us to final version:
```js
const preferredLanguage = process.env.LANG || 'en';
```

I'll refrain from making any additional comments.